packages/forms-web-app/src/routes/final-comment/check-your-answers.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+
+const checkYourAnswersController = require('../../controllers/final-comment/check-your-answers');
+const { validationErrorHandler } = require('../../validators/validation-error-handler');
+
+const router = express.Router();
+
+router.get('/check-your-answers', checkYourAnswersController.getCheckYourAnswers);
+router.post(
+	'/check-your-answers',
+	validationErrorHandler,
```

Comment:
is this handling anything? (usually the validation error handler has some validation rules middleware preceding it)